rviz_rendering/src/rviz_rendering/objects/billboard_line.cpp

```diff
@@ -185,6 +186,12 @@ void BillboardLine::addPoint(const Ogre::Vector3 & point, const Ogre::ColourValu
 
   rviz_rendering::MaterialManager::enableAlphaBlending(material_, color.a);
 
+  if (chain_containers_[current_chain_container_]->getNumChainElements(current_line_ %
+    chains_per_container_) >= max_points_per_line_)
```

Comment:
The `assert` above seems wrong: it should check for `<` (equivalent to this check), because if the values are equal before adding one, then adding 1 more will result in having too many elements. Maybe it's not worth changing it, though.